README.md

```diff
@@ -277,7 +277,7 @@ databases:
 
 ## Databases
 
-Right now, this app supports **MongoDB**, **PostgreSQL 13**, **MariaDB**, **Redis** and **Zip archiving**. If
+Right now, this app supports **MongoDB**, **PostgreSQL 13**, **MariaDB**, **Redis** and **LocalStorage archiving**. If
```

Comment:
this feels weird, and may be confused with `localStorage`, the HTML5 feature.

Let's just make it **local storage archiving** without capital letters.